pkg/personname/groupInfo.go

```diff
@@ -0,0 +1,170 @@
+package personname
+
+import (
+	"strings"
+)
+
+const segmentSep = "^"
+
+// GroupTrailingNullLevel represents how many null '^' separators are present in the
+// GroupInfo.DCM() return value.
+type GroupTrailingNullLevel uint
+
+// String implements fmt.Stringer, giving human-readable names to the trailing null
+// level.
+//
+// Returns "NONE" if no null separators were present.
+//
+// Returns "ALL" if the highest possible null separator was present.
+//
+// Otherwise, returns the name of the section that comes after the highest present null
+// separator.
+//
+// String will panic if called on a value that exceeds GroupNullAll.
+func (level GroupTrailingNullLevel) String() string {
+	switch level {
+	case GroupNullNone:
+		return "NONE"
+	case GroupNullGiven:
+		return "GivenName"
+	case GroupNullMiddle:
+		return "MiddleName"
+	case GroupNullPrefix:
+		return "NamePrefix"
+	case GroupNullAll:
+		return "ALL"
+	default:
+		panic(validateGroupNullSepLevel(level))
```

Comment:
Instead of panic, since this is a string representation being used most probably for debug, can we return something like "INVALID"?

Reply:
Hiya, I still see the Panic here--was it meant to be changed to "INVALID"?